Kitodo/src/main/java/org/kitodo/production/process/TitleGenerator.java

```diff
@@ -77,13 +77,28 @@ public String generateTitle(String titleDefinition, Map<String, String> genericF
     }
 
     /**
-     * Create Atstsl.
+     * Forms the author title key, or the title key (4/2/2/1) if no author is
+     * given. The author title key is a librarian sort criteria, composed out of
+     * the first four letters of the first author’s last name, followed by the
```

Comment:
how can it be formed out of the first author's last name if no author is given?

Reply:
If an author is specified, the author title key will be rendered. If no author is specified, the title key (4/2/2/1) is formed. Also, look at the examples below.